doc/backend_vars.asciidoc

```diff
@@ -7,6 +7,7 @@ Supported variables per backend
 |====================
 Variable;Values allowed;Default value;Explanation
 CASEDIR;string;;Path to test distribution. Can be a git repository URL of a test distribution to checkout with an optional refspec or also git hash into the current directory. It tries to follow the definition of https://docs.npmjs.com/files/package.json#git-urls-as-dependencies (e.g. `https://github.com/os-autoinst/os-autoinst-distri-opensuse.git#feature/test` or `git@github.com:os-autoinst/os-autoinst-distri-opensuse.git`). Note that cloning via SSH is only possible if git is configured accordingly on the host where isotovideo is executed. Can be combined with `TEST_GIT_REFSPEC` and `NEEDLES_GIT_REFSPEC`.
+DEFCASEDIR;string;;A default value of CASEDIR. When CASEDIR is a git repository URL of a test distribution, this value will be the path to test distribution. In this scenario it's used to find the correct needles directory.
```

Comment:
I haven't tested this scenario, I guess it does not work. I double if the current code does work for it without specifying NEEDLES_DIR. Because I do not change the needles_dir logic here, just search for more possible needles_dir.